src/platform_impl/windows/dark_mode.rs

```diff
@@ -72,9 +74,12 @@ lazy_static! {
 
 /// Attempt to set dark mode on a window, if necessary.
 /// Returns true if dark mode was set, false if not.
```

Comment:
The comment isn't not correct anymore, we now return `Theme`.

Reply:
fixed